x/gamm/pool-models/balancer/amm_test.go

```diff
@@ -14,6 +14,12 @@ import (
 	"github.com/osmosis-labs/osmosis/v9/x/gamm/types"
 )
 
+// allowedErrRatio is the maximal multiplicative difference in either
```

Comment:
@ValarDragon I wonder if we should require this to only be to the downside instead of in either direction since overestimates of DEX outputs are at disproportionately higher risk of being vulnerabilities?